packages/express-sample/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@logto/express-sample",
+  "version": "1.0.0-beta.0",
```

Comment:
better be `1.0.0-beta.3`?

Reply:
Nah, the latest version is `1.0.0-beta.2`, but it's also OK to use a smaller version number as it will be automatically bumped when publishing beta.3